client/my-sites/site-settings/site-icon-setting/index.jsx

```diff
@@ -256,8 +256,8 @@ class SiteIconSetting extends Component {
 				<FormLabel className="site-icon-setting__heading">
 					{ translate( 'Site Icon' ) }
 					<InfoPopover position="bottom right">
-						{ translate( 'Your site icon will be shown across WordPress.com, in your' +
-							' visitors\' browser tabs, and as a home screen app icon.' ) }
+						{ translate( 'The Site Icon is used as a browser and app icon for your site.' +
```

Comment:
Thanks! Let's match it as closely as possible, for the sake of translation and UX consistency.